src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = fromFormat.pop();
+  const newSeparator = toFormat.pop();
+  const dateParts = date.split(separator);
+  const newDate = ['', '', ''];
+
+  const formatMap = {
+    DD: 'день',
+    MM: 'місяць',
+    YYYY: 'рік',
+    YY: 'рік',
```

Comment:
The `formatMap` object contains values in a language other than English. It's important to maintain consistency and use English throughout the codebase unless the project specifications require otherwise.